src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -322,13 +323,15 @@ def _create_osd(self, svc_arg=None, inbuf=None):
 
         usage = """
 Usage:
-  ceph orchestrator osd create -i <json_file>
+  ceph orchestrator osd create -i <json_file/yaml_file>
   ceph orchestrator osd create host:device1,device2,...
 """
 
+        # TODO: try if inbuf file is yaml of json
```

Comment:
valid JSON is AFAIK also valid YAML. You can should be able to decode this with a pyyaml.

Reply:
idd